### PR TITLE
doc(readme): make it easier for users to understand why oomox is needed

### PR DESCRIPTION
**Pull request by actionless**
i am not sure if it make actually more clear or not :D 

but i was just thinking what after roundness changed in the new Materia release i can expect number of people asking "how to return old square buttons?" (the same as it was when default colors got changed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Materia (formerly Flat-Plat) is a [Material Design](https://material.io) theme for GNOME/GTK+ based desktop environments.  
 It supports GTK+ 3, GTK+ 2, GNOME Shell, Budgie, Cinnamon, MATE, Unity, LightDM, GDM, Chrome theme, etc.
 
-Also Materia has compatibility with [oomox theme designer](https://github.com/themix-project/oomox).
+Also Materia can be customized in [oomox theme designer](https://github.com/themix-project/oomox).
```
Comment (actionless) [inline README.md]: my english is not so good so i am actually not sure if it should be `in`, `with` or `using`

**Comment by nana-4** [comment]
Indeed, it sounds more clear to me. :)

File: README.md (hunk 1 above)
Comment (nana-4) [reply README.md]: My English is also not so good. :)

but I think `with` or `using` is correct (or better) in this case.

Can you update this to use `with`?

**Comment by nana-4** [comment]
LGTM. Thanks a lot! 🎉

**Pull request merged**